nomad/structs/structs.go

```diff
@@ -4457,6 +4457,19 @@ type Job struct {
 	ModifyIndex uint64
 	// JobModifyIndex is the index at which the job *specification* last changed
 	JobModifyIndex uint64
+
+	// Links and Description fields for the Web UI
+	Ui *JobUIConfig
```

Comment:
Idiomatic Go would have this field name in all-caps (https://go.dev/wiki/CodeReviewComments#initialisms):

```suggestion
	UI *JobUIConfig
```

That change should ripple down to the `api` package, docs, etc. as well.